src/server/server.cpp

```diff
@@ -1620,7 +1625,19 @@ void server::handle_player_in_game(socket_ptr socket, std::shared_ptr<simple_wml
 			const simple_wml::node::child_list& sides = g.get_sides_list();
 			for(unsigned side_index = 0; side_index < sides.size(); ++side_index) {
 				const simple_wml::node& side = *sides[side_index];
-				user_handler_->db_insert_game_player_info(uuid_, g.id(), side["player_id"].to_string(), side["side"].to_int(), side["is_host"].to_bool(), side["faction"].to_string());
+				const auto player = player_connections_.get<name_t>().find(side["player_id"].to_string());
```

Comment:
I wonder if the find call is necessary or this should iterate over something else.

It‘s just on game start though so shouldn’t be a big deal.

Reply:
I'm not sure where else I'd get the version/source info from, either.

Reply:
Getting the info from the player is fine, it’s just a bit weird to have to search. Could perhaps iterate over the players instead but maybe you‘d miss empty sides then or something. Haven’t looked more closely.

Reply:
Searching a multi-index by one of its indexed values should be pretty fast though, right?

Reply:
Yeah, in fact it should not be a search at all then. Maybe find works that way here. Still not very familiar with it.